### PR TITLE
Update with selene-sdk v0.2.0

### PR DESCRIPTION
**Pull request by kathyxchen**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by epruesse** [comment]
Whoops - sorry, I was too fast on #12632

**Comment by epruesse** [comment]
Would you like to add yourself as maintainer for the recipe (`extra: recipe-maintainers: - kathyxchen`)?

**Comment by kathyxchen** [comment]
Thanks @epruesse, I didn't know that was an option!

File: recipes/selene-sdk/meta.yaml [hunk 1]
```diff
@@ -35,11 +35,15 @@ requirements:
...
-    imports:
-        - selene_sdk
+  imports:
+    - selene_sdk
 
 about:
   home: https://github.com/FunctionLab/selene
   summary: Framework for developing sequence-level deep learning networks.
-  license: BSD 3-clause
+  license: BSD 3-clause clear
```
Comment (epruesse) [inline recipes/selene-sdk/meta.yaml]: clear?

Comment (kathyxchen) [reply recipes/selene-sdk/meta.yaml]: Yes, we've updated our license: https://github.com/FunctionLab/selene/blob/master/LICENSE

Comment (epruesse) [reply recipes/selene-sdk/meta.yaml]: Ah, no patent rights. Got it.

**Comment by epruesse** [comment]
Until recently, it didn't make a difference. Conda-Forge uses it to assign merge privileges to maintainers, we give those for everything to every team member.

I'm using it in `autobump` to know whom to @-mention in the generated PRs. So the only difference will be that next time you guys create a release, you'll be told about the automatically created PR.

**Pull request merged**